packages/std/src/init_handle.rs

```diff
@@ -81,6 +81,8 @@ pub enum WasmMsg {
         /// msg is the json-encoded InitMsg struct (as raw Binary)
         msg: Binary,
         send: Option<Vec<Coin>>,
+        /// optional human-readbale label for the contract
+        label: Option<String>,
```

Comment:
This is the label that already exists but now the instance gets informed about it?

Reply:
When coding the binding between cosmwasm messages and the sdk messages, I realized we didn't return label, which is a required field, so i gave some "auto-generated" message there. I figured it would be good to let the contract author expose this.

Note: I just added the ability for one wasm contract to *instantiate* another one in 0.8, which is why this didn't come up before. before we could only execute. (Instantiate makes sense to me when we start building 2 or 3 contracts that work together as a whole, so one can instantiate the sub-contracts it needs.